juju/model.py

```diff
@@ -43,32 +44,65 @@ def get_relation(self, relation_name, relation_id=None):
                 # errors, ideally integrating the error catching with Juju's mechanisms.
                 raise TooManyRelatedApps(relation_name, num_related, 1)
 
+    def _is_local_unit(self, unit_name):
+        return unit_name == self._local_unit_name
+
+    def _is_local_app(self, app_name):
+        return app_name == self.unit.app.name
```

Comment:
Per earlier PR, which had this embedded in it, these functions are not an improvement over the content they replace.